src/utils.js

```diff
@@ -845,3 +845,24 @@ export function unsupportedBidderMessage(adUnit, unSupportedBidders) {
     ${plural} won't fetch demand.
   `;
 }
+
+/**
+ * Delete property from object
+ * @param {Object} object 
```

Comment:
Not sure why Travis didn't catch this but when I check out the PR locally, getting a `no-trailing-spaces` lint error on this line

Reply:
Yep missed that. You found a bug in ESlint extension for VS Code